pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -244,22 +244,22 @@ func (z *zedrouter) init() (err error) {
 
 	z.peUsagePersist, err = persistcache.New(types.PersistCachePatchEnvelopesUsage)
 	if err != nil {
-		log.Fatal(err)
+		return err
```

Comment:
Is this a separate fix (which would make sense in a separate commit)?

Reply:
Yes, it is a minor unrelated fix, I will move it to a separate commit.

Reply:
Done